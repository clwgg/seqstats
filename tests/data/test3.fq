@r1
ACGT
+
!!!!
@r2
AC
+
!!
@r3
A
+
!
@r4
ACGTA
+
!!!!!

