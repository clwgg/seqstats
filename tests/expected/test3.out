Total n:	4
Total seq:	12 bp
Avg. seq:	3.00 bp
Median seq:	3.00 bp
N 50:		4 bp
Min seq:	1 bp
Max seq:	5 bp

